### PR TITLE
Add release task

### PR DESCRIPTION
**Pull request by pangratz**

### CODE REVIEW

**Comment by pangratz** [comment]
I'll adapt the RELEASING.md file ...

**Comment by dmathieu** [comment]
Thank you. I wrote the RELEASE.md file thinking of automating things, but I had social imperatives that made it impossible at that moment.
I'll merge this once you get the doc done :)

**Comment by pangratz** [comment]
One more thing: should the version in the `VERSION` file be updated to something like `0.0.5pre` after the version `0.0.4` has been updated?

File: RELEASING.md [hunk 1]
```diff
@@ -1,23 +1,9 @@
 # Releasing a new version
 
-Two steps are required to release a new version :
+A new version can be release by invoking `rake release`.
 
-* Tag it
-* Upload it
+The new version can be specified as an argument to the Rake task via `rake release[0.1.0]`. The supplied version number should be picker with full consideration of [semver](http://semver.org/).
```
Comment (dmathieu) [inline RELEASING.md]: Small typo: picker -> picked.
You could also make it in two lines (but that's neatpicking).

**Comment by dmathieu** [comment]
Yup, that'd be cool. The tagging should happen in the commit before upgrading the version though.

**Comment by pangratz** [comment]
But currently the version specified in `VERSION` represents the last released version, right?

**Comment by dmathieu** [comment]
Well currently, there is no official release yet.
I released something as 0.0.1 4 days ago, but it was just to facilitate the use of the library.
I intend to release 1.0.0 around the same time as emberjs 1.0 is released.

**Comment by pangratz** [comment]
OK. I was just checking at the Ember.js repo and the VERSION currently is `1.0pre`. In my understanding, as soon as something is released (aka a file with the version is available), this version should not be used again.

But this is just a problem of definition. I'll update the PR so the version in the `VERSION` file is the currently developed version, which will be released next...

**Comment by dmathieu** [comment]
Yes, that's how it should be. Thanks.

File: Rakefile [hunk 2]
```diff
@@ -26,6 +26,32 @@ def upload_file(uploader, filename, description, file)
...
+  version = File.open("VERSION", "rb").read.strip!
+  Versionomy.parse(version).bump(:tiny)
+end
+
+desc "Release current source and bump to new version"
+task :release, [:new_version] => :dist do |t, args|
+  # Release this version
+  version = File.open("VERSION", "rb").read.strip!
+  puts "Releasing new version #{version}"
+  system "git commit VERSION -m 'Release new version v#{version}'"
```
Comment (dmathieu) [inline Rakefile]: I don't think this commit is useful, as the VERSION file shouldn't have changed. So it wouldn't happen.

Comment (pangratz) [inline Rakefile]: You're absolutely right! Good catch.

**Comment by dmathieu** [comment]
Awesome ! Thanks.

**Pull request merged**